app.go

```diff
@@ -14,9 +15,16 @@ import (
 func app(cmd *cobra.Command, args []string) {
 	var version string
 	var err error
+	platform, err := GetOperatingSystem()
+	if err != nil {
+		glog.Errorf("Failed to get operating system platform: %v", err)
+	}
+	platform = strings.TrimSpace(platform)
```

Comment:
looks like `TrimSpace` is common behavior for GetOperatingSystem, no?
I can miss something here